doc/topics/blocking-vs-non-blocking.md

```diff
@@ -0,0 +1,140 @@
+# Overview of Blocking vs Non-Blocking
+
+This overview covers the difference between blocking and non-blocking calls in
```

Comment:
hmm, we usually reserve the code `highlighting` for code or literal values. It may be confusing. Perhaps just use **bold**?